crates/air-lib/test-utils/src/executed_state.rs

```diff
@@ -132,21 +132,12 @@ pub fn subtrace_desc(begin_pos: impl Into<TracePos>, subtrace_len: u32) -> SubTr
     }
 }
 
-pub fn ap(dst: Option<u32>) -> ExecutedState {
-    let res_generations = option_to_vec(dst);
-    let ap_result = ApResult::new(res_generations);
-
+pub fn ap(generation: u32) -> ExecutedState {
+    let ap_result = ApResult::new(vec![generation]);
```

Comment:
The `ApResult::new` can be further refactored to accept a single number: 4136257804228ba6e69f626739c5e6917f3ad0c9

Reply:
Yep, will change it.